.yarn/versions/ce619930.yml

```diff
@@ -0,0 +1,43 @@
+releases:
+  "@radix-ui/react-accessible-icon": patch
```

Comment:
Just checking if you've re-run this from scratch after the updates on the packages?